README.md

```diff
@@ -20,6 +20,9 @@ Additional description about the project and its features.
 # Live Demo Of The Project
 https://alexander16108.github.io/destop-version/
 
+# Online version of this project 
+https://alexander16108.github.io/
+
 # Usage
 This is used for learning purposes. Feel free to collaborate.
 
```

Comment:
- [ ] Kindly update your readme to include instructions on how to clone the project and run it locally. 
- [ ] Also, you have added 2 live links in your project, remove one of them.
- [ ] The deployment section of your readme is not necessary. Contributing to the project should not be put under the deployment header as well.
- [ ] You can merge the Author and social media sections together. For example,

> Author
>  Alex Odufuye 
>  - Github: @alexander16108